contracts/WildcardSteward_v3.sol

```diff
@@ -165,14 +166,22 @@ contract WildcardSteward_v3 is Initializable {
         require(_newPrice > 0, "Price is zero");
         _;
     }
+    modifier notNullAddress(address checkAddress) {
+        require(checkAddress != address(0), "null address");
+        _;
+    }
+    modifier notSameAddress(address firstAddress, address secondAddress) {
+        require(firstAddress != secondAddress, "cannot be same address");
+        _;
+    }
```

Comment:
These two checks were in two functions, so I thought it made sense to make them modifiers.